### ISSUE TITLE
Is baseline methods available?

### ISSUE THREAD

**Issue by zjysteven** [avduarte333/DE-COP_Method#1]
Hi,

Thanks for releasing the datasets. I was wondering if it is possible to also release your implementation of applying those baselines (e.g., perplexity, Min-K%) to BookTection. This will be very helpful since I'm trying to apply my method (which has some similarity to Min-K%) to BookTection but has no idea how to get started. I believe this will also benefit wider adoption of BookTection if people can quickly test their methods on the deatset.

Thanks

**Comment by avduarte333** [comment]
Hey,
We mostly just used the implementation that the authors of min-k-prob provide in their [GitHub](https://github.com/swj0419/detect-pretrain-code/blob/main/src/run.py). Nonetheless, if uploading our script makes it easier to run a comparison on BookTection, we'll try to put this here on the following days.

**Comment by zjysteven** [comment]
Yes, that would be great. I'm just unsure about some details when applying Min-K to a chat model and to QA based input.

**Comment by avduarte333** [comment]
Oh, maybe let me explain this a bit better: so Min-K and the baselines for the models with logits access, are applied to the base versions of the models! Like [LLaMA-2 7B](https://huggingface.co/meta-llama/Llama-2-7b-hf)

We thought this would be more appropriate because, since calculating perplexity or any of the other baselines is just basically, evaluate the model's ability to predict general language structures, the base version would be a better fit.
Only DE-COP works on the chat-version because it's task envolves reasoning and is more aligned to a human conversation.

**Comment by zjysteven** [comment]
That makes sense. Thanks for the clarification!
